### PR TITLE
Declare option for legacy format

### PR DESCRIPTION
**Pull request by hmusum**

### CODE REVIEW

**Review by aressem — APPROVED** [review]

File: config/src/apps/vespa-get-config/getconfig.cpp [hunk 1]
```diff
@@ -119,7 +119,7 @@ GetConfig::Main()
 
     const char *optArg = NULL;
     int optInd = 0;
-    while ((c = GetOpt("a:n:v:i:jm:c:t:V:w:r:s:p:dh", optArg, optInd)) != -1) {
+    while ((c = GetOpt("a:n:v:i:jlm:c:t:V:w:r:s:p:dh", optArg, optInd)) != -1) {
```
Comment (arnej27959) [inline config/src/apps/vespa-get-config/getconfig.cpp]: this looks wrong, now 'j' will require and argument.  should be ..."i:jlm:"...

**Review by arnej27959 — CHANGES_REQUESTED** [review]
must fix GetOpt syntax

File: config/src/apps/vespa-get-config/getconfig.cpp (hunk 1 above)
Comment (hmusum) [reply config/src/apps/vespa-get-config/getconfig.cpp]: Yeah, I saw and fixed that some minutes ago

**Review by arnej27959 — APPROVED** [review]
looks good now

**Pull request merged**